server/src/main/java/org/elasticsearch/cluster/metadata/DataStreamFailureStoreDefinition.java

```diff
@@ -131,4 +134,44 @@ public class DataStreamFailureStoreDefinition {
             throw new AssertionError(e);
         }
     }
+
+    public static final String FAILURE_STORE_REFRESH_INTERVAL_SETTING_NAME = "data_streams.failure_store.refresh_interval";
```

Comment:
Nit: I am personally more a fan of grouping fields of certain types (static, final, etc.) together. I get that this is a bit of an exceptional case as we initialize the `DATA_STREAM_FAILURE_STORE_MAPPING` field in a `static {}` block, but I'd still be in favor of moving `FAILURE_STORE_REFRESH_INTERVAL_SETTING_NAME` to the top of the class. I usually find it confusing to read a class where fields are declared in different places.